homeassistant/components/coinbase/const.py

```diff
@@ -262,10 +273,13 @@
     "XPT": "XPT",
     "XRP": "XRP",
     "XTZ": "XTZ",
+    "XYO": "XYO",
     "YER": "YER",
     "YFI": "YFI",
+    "YFII": "YFII",
```

Comment:
```suggestion
```

Yearn Finance II (YFII) is also not tradeable on Coinbase so again isn't a valid wallet, https://www.coinbase.com/price/yearn-finance-ii